server/helpers/identification.js

```diff
@@ -0,0 +1,25 @@
+import { UserRole, User, Role } from '../database/models';
+import responseMessage from './responseMessage';
+
+const getUserId = async (email, response) => {
+  try {
+    const { dataValues } = await User.findOne({ where: { email } });
+    return dataValues.id;
+  } catch (error) {
+    responseMessage(response, 404, { message: 'Account not found' });
+  }
+};
+const getUserRole = async (param, response) => {
+  try {
+    const userId = (/\D/g.test(param)) ? await getUserId(param, response) : param;
+    const { dataValues } = await UserRole.findOne({
+      where: { userId },
+      include: [{ model: Role, required: true }]
+    });
+    return responseMessage(response, 200, { data: dataValues.Role.dataValues }, 'success');
```

Comment:
Add the status to the object. Don't pass it as a fourth argument.

Reply:
If I add it to the object, it will show the failure status first

Reply:
No, it will override the failure status already there. Just put it as the first property in the object

Reply:
Done